From 0ffb3aca8989f6243e892d4cab9306520533a5e5 Mon Sep 17 00:00:00 2001
From: Tony Cheng <tony.cheng@amd.com>
Date: Thu, 25 Aug 2016 23:33:36 -0400
Subject: [PATCH 0944/1722] drm/amd/dal: remove oem_ddc from vbios.

In dal2 oem ddc is used only from escape DeviceEscapeType_OEM.
Since atomfirm stop supporting oem_ddc it's safe to assume that we can say we
don't support oem_ddc in dal3

Change-Id: Ia6a0978db2650d20ad15c974c0cc8223cd3f541a
Signed-off-by: Jordan Lazare <Jordan.Lazare@amd.com>
Acked-by: Harry Wentland <harry.wentland@amd.com>
Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c | 4 ----
 drivers/gpu/drm/amd/dal/dc/dc_bios_types.h    | 5 -----
 2 files changed, 9 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c b/drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c
index a71acbf..60b5ceb 100644
--- a/drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c
+++ b/drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c
@@ -4364,8 +4364,6 @@ static void bios_parser_destroy_integrated_info(
 static const struct dc_vbios_funcs vbios_funcs = {
         .get_connectors_number = bios_parser_get_connectors_number,
 
-        .get_oem_ddc_lines_number = bios_parser_get_oem_ddc_lines_number,
-
         .get_encoder_id = bios_parser_get_encoder_id,
 
         .get_connector_id = bios_parser_get_connector_id,
@@ -4380,8 +4378,6 @@ static const struct dc_vbios_funcs vbios_funcs = {
 
         .get_i2c_info = bios_parser_get_i2c_info,
 
-        .get_oem_ddc_info = bios_parser_get_oem_ddc_info,
-
         .get_voltage_ddc_info = bios_parser_get_voltage_ddc_info,
 
         .get_thermal_ddc_info = bios_parser_get_thermal_ddc_info,
diff --git a/drivers/gpu/drm/amd/dal/dc/dc_bios_types.h b/drivers/gpu/drm/amd/dal/dc/dc_bios_types.h
index e928419..1c437d3 100644
--- a/drivers/gpu/drm/amd/dal/dc/dc_bios_types.h
+++ b/drivers/gpu/drm/amd/dal/dc/dc_bios_types.h
@@ -40,7 +40,6 @@
 
 struct dc_vbios_funcs {
 	uint8_t (*get_connectors_number)(struct dc_bios *bios);
-	uint32_t (*get_oem_ddc_lines_number)(struct dc_bios *bios);
 
 	struct graphics_object_id (*get_encoder_id)(
 		struct dc_bios *bios,
@@ -66,10 +65,6 @@ struct dc_vbios_funcs {
 		struct dc_bios *bios,
 		struct graphics_object_id object_id, uint32_t index,
 		struct graphics_object_id *dest_object_id);
-	enum bp_result (*get_oem_ddc_info)(
-		struct dc_bios *bios,
-		uint32_t index,
-		struct graphics_object_i2c_info *info);
 
 	enum bp_result (*get_i2c_info)(
 		struct dc_bios *dcb,
-- 
2.7.4

